docs/tutorials/gettingstartedexample/gse-gen-common.md

```diff
@@ -16,16 +13,133 @@ This document covers a range of common scenarios using the PXIe-148X Generation
+    | Resource | TDMS File Directory| A directory path to TDMS files containing LLP Packet data for serial output channels (e.g. <font face = "courier new">SI0_LLP_Packets.tdms</font>) |
+    
+    > Note: The **TDMS File Directory** is a folder selection and the browse dialog shows folders only, not file names. 
+    >
+    > Use the default **TDMS File Directory** value if generating from a TDMS file created with the Create CSI-2 Packet TDMS Files utility's default settings. See the [PXIe-148X Getting Started Example - Basic Generation Tutorial](gse-gen-basic.md#create-tdms-files-for-generation) for details on creating TDMS files.
+
+4. Double click the Acquisition Example VI in the acquisition LabVIEW project.
+
+5. Set the following controls on the Acquisition Example GSE VI and leave all other values at their defaults.
+    > Note: VI controls and indicators can be reset to default values by clicking on the **Edit** menu and selecting the **Reinitialize Values to Default** option.
+
+    | Tab            | Control                 | Value                                                                                                                                                               |
+    |----------------|-------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+    | Resource       | RIO Device              | [System Specific]                                                                                                                                                   |
+    | Resource       | Bitfile Path            | [Refer to Bitfile Path in the PXIe-148X Acquisition GSE Help](../../reference/gettingstartedexample/gse-acq-help.md#table-of-pxie-148x-acquisition-bitfiles)        |
+    | Resource       | Log Packets to Disk     | Enabled                                                                                                                                                             |
+    | Serial Channel | Configuration Script    | [Refer to Configuration Script in the PXIe-148X Acquisition GSE Help](../../reference/gettingstartedexample/gse-acq-help.md#table-of-pxie-148x-acquisition-scripts) |
+    | Acquisition    | Acquisition Duration    | 5 seconds                                                                                                                                                           |
+    | Acquisition    | Continuous Acquisition  | Enabled                                                                                                                                                            |
+    | Board          | Power Over Coax Source  | Internal   
+
+6. Run the Acquisition Example VI.
+
+7. Run the Generation Example VI.
```

Comment:
Althought this works with the default loopback script I think it would be worth considering flipping the order of these steps and adding another step at the end explicitly telling the user to click Serializer Setup Complete. (Adds consistency with the phrasing used in Jon's tutorial portions and with the Basic tutorial, and reinforces the workflow that is needed when the script has I2c commands addressing the serializer)

e.g.
- Run the Generation Example VI and wait for the Waiting for Serializer Setup indicator to illuminate.
- Run the Acquisition Example VI.
- After the Acquisition Example VI indicates acquisition has started, click Serializer Setup Complete on the Generation VI before the Acquisition VI stops.